fluster/decoders/ffmpeg.py

```diff
@@ -27,8 +27,19 @@
 from fluster.decoder import Decoder, register_decoder
 from fluster.utils import file_checksum, run_command
 
-FFMPEG_TPL = '{} -i {} {} -vf format=pix_fmts={} -f rawvideo {}'
-
+FFMPEG_TPL = '{} -i {} {} -vf {}format=pix_fmts={} -f rawvideo {}'
+
+def output_format_to_ffformat(output_format: OutputFormat) -> str:
+    """Return GStreamer pixel format"""
+    mapping = {
+        OutputFormat.YUV420P: "nv12",
+        OutputFormat.YUV422P: "nv12", # vulkan
+        OutputFormat.YUV420P10LE: "p010",
+        OutputFormat.YUV422P10LE: "p012",
+    }
+    if output_format not in mapping:
+        raise Exception(f"No matching output format found in FFMpeg for {output_format}")
```

Comment:
```suggestion
        raise Exception(f"No matching output format found in FFmpeg for {output_format}")
```

I see `FFMpeg` on many sites, but I don't know what the origin is. I think correct is `FFmpeg`.

Reply:
thanks :)